README.md

```
@@ -0,0 +1,243 @@
+That's it. The app shows progress in real-time and you can cancel anytime.
+
+## A few things to know
+
+- It tries to download the exact resolution you pick. If that's not available, it falls back to the next best thing.
+- Videos get merged into a single MP4 file with H.264 codec for compatibility.
+- Audio gets extracted in whatever format you chose.
+- Download history is saved locally so you can find your files later.
+=======
+# Predator - YouTube Downloader
+<p align="center">
+<img src="asset/Predatorv1.png">
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)